cmd/bosun/expr/funcs.go

```diff
@@ -355,7 +364,7 @@ func parseGraphiteResponse(req *graphite.Request, s *graphite.Response, formatTa
 		// build tag set
 		nodes := strings.Split(res.Target, ".")
 		if len(nodes) < len(formatTags) {
-			return nil, fmt.Errorf(`returned target "%s" does not match format "%s"`, res.Target, formatTags)
+			return nil, graphite.NewError(req.String(), "", fmt.Errorf(`returned target "%s" does not match format "%s"`, res.Target, formatTags))
```

Comment:
i want to make this back to a regular error but for the reason that if there's no body (==extra info for log) than there's no need to use graphite.Error.
I do disagree that graphite.Error should only be used for web errors. they are designed to be used any time you want to print extra stuff to the log.